Products/RhaptosModuleStorage/ModuleDBTool.py

```diff
@@ -231,7 +231,7 @@ def _getFileID(self,fileob):
             if r.file == fdata:
                 return r.fileid
         # Fell through, must be new bytes
-        res = self.sqlInsertFile(file = Binary(fdata))
+        res = self.sqlInsertFile(file = Binary(fdata), media_type=fob.content_type)
```

Comment:
Yeah, I believe you are correct. The variable looks to be passed in as ``fileob``.